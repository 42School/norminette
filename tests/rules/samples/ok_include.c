#include <stdio.h>
#include "main.h"

#include <main.h>