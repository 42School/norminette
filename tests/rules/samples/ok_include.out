[36mok_include.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 1":
		<INCLUDE=#include <stdio.h>> <NEWLINE>
[36mok_include.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 2":
		<INCLUDE=#include "main.h"> <NEWLINE>
[36mok_include.c[0m - [32mIsEmptyLine[0m In "GlobalScope" from "None" line 3":
		<NEWLINE>
[36mok_include.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 4":
		<INCLUDE=#include <main.h>> 
ok_include.c: Error!
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
